utils/pip/pipdeptree/scriptCreator.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	pipDepTreeVersion             = "3"
```

Comment:
I suggest adding the following comment above this line:
// The pip-dep-tree script version. The version should be manually incremented following changes to the `pipdeptree.py` source file.